primitives/src/sybil.rs

```diff
@@ -0,0 +1,167 @@
+// Copyright (c) 2019 Alain Brenzikofer
```

Comment:
This file has been removed on master and is probably a rebase error

Reply:
Yep, fixed in 5cd2060.